wallettool/build.gradle

```diff
@@ -48,6 +52,10 @@ if (hasJunit5) {
     }
 }
 
+run {
+    args "--wallet=test.wallet", '--help'
```

Comment:
Is there a reason you're providing those two parameters at the same time? It doesn't fail, but I think providing a wallet doesn't make sense in the `--help` case. I think the wallet will be ignored.

Reply:
To be honest, the `run` task doesn't really even need to be there. It was kind of a local modification I was using for convenience. I'll force-push an update with `run` removed entirely.

Reply:
I'm removing the entire `run` closure.